src/style.css

```diff
@@ -1,3 +1,78 @@
 body {
   margin: 0;
+  text-transform: uppercase;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: white;
+  width: 100% - 100px;
+  padding: 0 50px;
+}
+
+.logo__img {
+  display: block;
+}
+
+.nav {
+  text-align: center;
+  display: inline-block;
+}
+
+.nav__link {
+  color: black;
+  text-decoration: none;
+  display: inline-block;
+  line-height: 60px;
+}
+
+.nav__link__margin {
+  margin: 0 10px;
+}
+
+.nav__link__margin__right {
+  margin-right: 10px;
+}
+
+.nav__link__margin__left {
+  margin-left: 10px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  justify-content: space-between;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__link__margin__apple {
+  margin-right: 10px;
+}
+
+.nav__link__margin__video {
+  margin-left: 10px;
+}
```

Comment:
it is last-child, so just remove margin for last-child, not necessary to create additional class